src/test/java/com/team3/central/services/OrganizerSerivceTest.java

```diff
@@ -1,5 +1,144 @@
 package com.team3.central.services;
 
-public class OrganizerSerivceTest {
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.Mockito.anyString;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.team3.central.repositories.OrganizerRepository;
+import com.team3.central.repositories.SessionTokenRepository;
+import com.team3.central.repositories.entities.ConfirmationToken;
+import com.team3.central.repositories.entities.OrganizerEntity;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+class OrganizerServiceTest {
+  private OrganizerService organizerService;
+  @Mock
+  private OrganizerRepository organizerRepository;
+  @Mock
+  private SessionTokenRepository sessionTokenRepository;
+  @Mock
+  private BCryptPasswordEncoder bCryptPasswordEncoder;
+  @Mock
+  private ConfirmationTokenService confirmationTokenService;
+  @Mock
+  private EmailService emailService;
+
+  @BeforeEach
+  void setUp() {
+    MockitoAnnotations.openMocks(this);
+    organizerService = new OrganizerService(organizerRepository, sessionTokenRepository,
+        bCryptPasswordEncoder, confirmationTokenService, emailService);
+  }
```

Comment:
Set up as @BeforeAll with static method instead (unless you need to change this mocks and service everytime)